src/Compilers/VisualBasic/Portable/Lowering/Rewriter.vb

```diff
@@ -118,6 +119,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Try
         End Function
 
+        <SuppressMessage("Style", "VSTHRD200:Use ""Async"" suffix for async methods", Justification:="'Async' refers to the language feature here.")>
```

Comment:
Why an attribute rather than `#Disable Warning`?

Reply:
1. It's one line instead of two (the `#Disable Warning` would still go on this line, but then it would require another line after the method header)
2. Easy to include the rationale

Reply:
The suppression is only needed in source, not from metadata.

---
In reply to: [452462415](https://github.com/dotnet/roslyn/pull/45628#discussion_r452462415) [](ancestors = 452462415)

Reply:
`SuppressMessage` is a conditional attribute. I don't believe we are defining `CODE_ANALYSIS` in our production builds so both forms would be source-only suppressions.